### DISCUSSION TITLE
Save both original .png image and semantic segmentation .png mask in manual_control.py

### DISCUSSION THREAD

**Discussion by jevrub** [carla-simulator/carla#6348] [Q&A]
Hi,

I want to integrate the functionality to save two types of images at the same time into the existing CARLA script **_manual_control.py_**
When recording is ON, I need to save to disk
1) original png image (it is already implemented in **_manual_control.py_**
2) semantic segmentation mask (need to develop)

**Note: both images should be taken at the same without any latency**

I added this part inside manual_control.py:
`image.save_to_disk('_out/s%08d' % image.frame, carla.ColorConverter.CityScapesPalette)`

![image](https://user-images.githubusercontent.com/85548142/227989217-415dc04e-f4d4-403f-8b12-f7b776981caf.png)

But this leads to the obviously incorrect semantic segmentation mask.
![s00008803](https://user-images.githubusercontent.com/85548142/227988467-e9e9b1e0-1edb-44ec-9016-94fa663490b4.png)

I need to get normal segmentation mask, for example:
![o00077069](https://user-images.githubusercontent.com/85548142/227989354-4240a4f1-ddbe-4b6e-a245-ddfb992ca001.png)

**Comment by anja-sheppard** [comment]
You need to create another CARLA camera and mount it in the same location.

```
camera_bp = blueprint_library.find('sensor.camera.instance_segmentation')
camera_transform = carla.Transform(carla.Location(x=1.5, z=2.4))
camera_ground = world.spawn_actor(camera_bp, camera_transform, attach_to=vehicle)

camera_ground.listen(lambda image: image.save_to_disk('_out/ground/%06d_is_1.png' % image.frame, cc_is_1))
```

Also in the future don't attach a screenshot of your code--paste in the text.
